department-of-veterans-affairs/caseflow
# adding new bulk scripts

Resolves [APPEALS-33934](https://jira.devops.va.gov/browse/APPEALS-33934)

# Description
Consolidate the make audit script into 3 simple scripts.

## Acceptance Criteria
- [ ] make reset should take a little less time.
- [ ] No spec failure should be a result. 

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Run Make reset and see if it fails somewhere


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
